packages/snage/src/config/load.ts

```diff
@@ -5,9 +5,56 @@ import {parseConfig} from './parser';
 import {Config, Field, hasProvided, RawConfig, RawField} from './type';
 import * as E from 'fp-ts/lib/Either';
 import * as A from 'fp-ts/lib/Array';
+import * as O from 'fp-ts/lib/Option';
+import * as IO from 'fp-ts/lib/IO';
+import * as IOE from 'fp-ts/lib/IOEither';
 import {pipe} from 'fp-ts/lib/pipeable';
 import {getValueProvider} from '../provider/provider';
 import {getOrdering} from '../query/sort';
+import {ConfigParameterName, printAndExit} from '../command/common';
+import {flow} from 'fp-ts/lib/function';
+
+export const getConfig = (): E.Either<string, Config> => pipe(getConfigFile()(), E.chain(loadConfig));
+
+export const getConfigOrExit = (): Config => {
```

Comment:
Prinzipiell sind das ja alles IO operationen, die frage ist, ob wir das auch so definieren wollen:
```suggestion
export const getConfigOrExit: IO.IO<Config> = () => {
```

Reply:
Bin mir auch unsicher. Idealerweise sollte alles was I/O macht auch in IO leben, also sowas wie `readFileSync` und Co. Und dann sollte das nie wieder aus IO escapen können bis `main` analog zu `Task`. Aber das macht den Code teilweise ziemlich unlesbar, so dass es vielleicht keine besonders gute Idee ist.

Reply:
Ich entferne die ganzen IO-Typen erst mal wieder aus `load.ts`. So wie ich das aktuell verwende, nutzt das eher nichts und schadet nur der Lesbarkeit, glaube ich.